### ISSUE TITLE
gel bomb and and gel streams dont respect gel colors

### ISSUE THREAD

**Issue by Hikkikira** [StrataSource/Portal-2-Community-Edition#1354]
### Describe the bug

paint does not respect the player's gel color choice
![sp_a3_speed_flings0001](https://github.com/StrataSource/Portal-2-Community-Edition/assets/111175647/808cbe07-7d6e-400e-8dba-e70a5cb7f221)
![sp_a3_speed_flings0002](https://github.com/StrataSource/Portal-2-Community-Edition/assets/111175647/ac2d45c4-291c-4a9d-b8ea-f1a7611c8a63)

### To Reproduce

change gel color with one of the respective commands and observe

**Comment by Luke18033** [comment]
This is known, there's a bit of work that's been done to fix it but it's not finished yet.

**Comment by Hikkikira** [comment]
just make sure the blob_curve particles obey player color too, ( or maybe make new ones named after each gel? (blob_curve_bounce, blob_curve_stick, blob_curve_speed, blob_curve_portal, blob_curve_reflect, blob_curve_erase))

**Comment by AWildErin** [comment]
There won’t be multiple particles for this. Plan is to just have 1 set which sets the colour as needed directly from the paint color cvar

**Comment by Hikkikira** [comment]
blob_curve is for info_particle_systems

**Comment by AWildErin** [comment]
Right, and I'm saying that there won't be multiple particles for this. As the plan is to allow for custom paints in the future, adding specific ones for specific paints is not a good idea, nor does it solve the issue. The plan is to actually make them respect the cvar as particles already have the ability to set their colour, they just use materials for it

**Comment by Hikkikira** [comment]
thanks? i'm assuming this ,eams info_particle_system will have a new keyvaule?
